pythonosc/osc_message.py

```diff
@@ -49,11 +52,25 @@ def _parse_datagram(self):
           val = True
         elif param == "F": # False.
           val = False
+        elif param == "[": # Array start.
+          a = []
```

Comment:
please avoid single letter vars, at least 3 like "arg"/"val" is the minimum acceptable ;)